infra/es_cluster.tf

```diff
@@ -6,17 +6,27 @@ resource "google_compute_network" "es" {
 }
 
 locals {
+  es_ssh  = 0
+  es_feed = 1
   es_clusters = [
     {
       suffix         = "-blue",
       ubuntu_version = "2004",
-      size           = 5,
+      size           = 10,
+      init           = "[]",
     },
     {
       suffix         = "-green",
       ubuntu_version = "2004",
       size           = 0,
-    }
+      init           = "[]",
+    },
+    {
+      suffix         = "-init",
+      ubuntu_version = "2004",
+      size           = 0,
```

Comment:
I’m confused, what does size 0 do?

Reply:
That's the number of machines. In normal operation, you don't want any machine with an initial_master set. So this is there only as a knob to turn manually if/when reinitializing the cluster.